### PR TITLE
Update docker and contribution docs

### LINKED ISSUE #212: Create a robust contribution.md file
Example:
https://github.com/atom/atom/blob/master/CONTRIBUTING.md

Guidelines:
https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

### PR DESCRIPTION
**Pull request by seanpmorgan**
Closes #212

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -1,55 +1,98 @@
...
-NOTE: if you're a developer working on armory from the master branch, you will need to 
-build the dev containers: 
+## Docker Volume Mounts
+When launching an ARMORY instance several host directories will be mounted within the 
+docker container:
+
+| Host Path   | Docker Path   | 
+|:----------: | :-----------: | 
+| os.getcwd() | /workspace    |  
+| ~/armory/datasets | /armory/datasets |
```
Comment (ng390) [inline docs/docker.md]: Should this be ~/.armory/datasets, etc, or <host armorydir>/datasets?

Comment (seanpmorgan) [reply docs/docker.md]: So below I note that those are the default values. @davidslater any thoughts on which way to show it? I'm fine either way just looking for another set of eyes

Comment (seanpmorgan) [reply docs/docker.md]: Good catch. Updated.

**Comment by ng390** [comment]
LGTM.

**Pull request merged**